tests/integration/test_pip_packages.py

```diff
@@ -77,12 +137,13 @@ def assert_packages_from_response(response_data, expected_packages):
         assert expected_pkg in packages
 
 
-def assert_expected_files(source_path, expected_file_urls=None):
+def assert_expected_files(source_path, expected_file_urls=None, check_content=True):
```

Comment:
Should this change be part of this commit?

Reply:
@nirzari I added it as a new commit before the main one. Thank you!